src/main/kotlin/de/bigboot/ggtools/fang/service/MistforgeService.kt

```diff
@@ -32,21 +33,21 @@ interface MistforgeComponentSpec : ComponentSpec {
     }
 }
 
-data class ButtonBuildDetails(val buildId: String): MistforgeComponentSpec {
-    override fun id() = "$PREFIX:${buildId}"
+data class ButtonBuildDetails(val params: String): MistforgeComponentSpec {
```

Comment:
Make 2 Buttons instead, one with buildId and another one with heroId,skills,talens and summons

Reply:
If you make it 2 different buttons wont you have to code the reply message 2 times? Or would it be possible to make a function that returns [this](https://github.com/BigBoot/fang/blob/6d87ba79788da4a555286d84401f8fae0323ff76/src/main/kotlin/de/bigboot/ggtools/fang/service/MistforgeService.kt#L231-L256)?

Reply:
You could for example split the handler into multiple functions, or make the buttons sealed and handle them with `when`, see https://kotlinlang.org/docs/sealed-classes.html